ibm/mas_devops/playbooks/dependencies/install-db2.yml

```diff
@@ -0,0 +1,16 @@
+---
+- hosts: localhost
+  any_errors_fatal: true
+  vars:
```

Comment:
Can all of these vars be removed to align with the newer style of playbook where we pushed all the lookups/defaults into the role's defaults/main.yml so that they are all in one place inside the role?

Looing at the list of vars I think all of these are covered by existing entries in the role already.

Reply:
Removed the vars and updated the playbook dependencies doc to show what environment variables are required.